backend/app/tests/api/routes/test_openai_conversation.py

```
@@ -0,0 +1,267 @@
+        user_question="What is the capital of France?",
+        assistant_response="The capital of France is Paris.",
+        model="gpt-4o",
+        input_tokens=10,
+        output_tokens=5,
+        total_tokens=15,
+        assistant_id="asst_123",
+        project_id=1,
+        organization_id=1,
+    )
+    conversation = create_openai_conversation(db, conversation_data)
+    headers = {"X-API-KEY": original_api_key}
```

Ruff (0.12.2)

Undefined name `original_api_key`

(F821)